medusa/search/core.py

```diff
@@ -830,13 +830,12 @@ def search_providers(series_obj, episodes, forced_search=False, down_cur_quality
                     if ep_obj.episode in found_results[cur_provider.name]:
                         log.debug(
                             u'A needed multi-episode result overlaps with a single-episode result for episode {0},'
-                            u' removing the single-episode results from the list',
+                            u' replacing the single-episode results from the list',
                             ep_obj.episode,
                         )
-                        del found_results[cur_provider.name][ep_obj.episode]
+                    found_results[cur_provider.name][ep_obj.episode] = multi_result
```

Comment:
I'm not sure, but could this cause the same `multi_result` to appear multiple times (once for each episode)?

Reply:
In theory that's what this code should prevent: https://github.com/pymedusa/Medusa/blob/3a769ad89776bf9b40f3905539725bc8237f5140/medusa/search/core.py#L788-L825

Not sure if it actually works tho.

Reply:
I read through the code, and I actually think it's [this code](https://github.com/pymedusa/Medusa/blob/d39c1ba/medusa/search/core.py#L855-L867).
Anyway I think that part is fine.
Now I'm not so sure if the change will actually fix anything. :thinking: